.devcontainer/3ds2-example/devcontainer.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "Adyen 3DS2 Example",
+  "image": "mcr.microsoft.com/devcontainers/universal:2",
+  "postAttachCommand": "bash ../.devcontainer/setup.sh",
```

Comment:
Change this to: https://github.com/adyen-examples/adyen-node-online-payments/pull/940/files#r2610452671

By adding the script here